pkg/provider/gitea/gitea.go

```diff
@@ -135,6 +142,10 @@ func (v *Provider) createStatusCommit(event *info.Event, pacopts *info.PacOpts,
 	switch status.Conclusion {
 	case "skipped", "neutral":
```

Comment:
do we still have this case anymore? I see you updating this case.

Reply:
Its not needed now 
Removed